gtsam_unstable/gtsam_unstable.i

```diff
@@ -163,6 +163,15 @@ class BearingS2 {
   void serializable() const; // enabling serialization functionality
 };
 
+#include <gtsam_unstable/geometry/Similarity3.h>
+class Similarity3 {
+  Similarity3();
+  Similarity3(double s);
+  Similarity3(const gtsam::Rot3& R, const gtsam::Point3& t, double s);
+  Similarity3(const Matrix& R, const Vector& t, double s);
+  Similarity3(const Matrix& T);
```

Comment:
Do these constructors check the size of the matrix?

Reply:
They don't. In the whole gtsam.i file actually only `Matrix & T` or `Vector & t` is used, and `Matrix3` etc throws a compilation error